packages/ra-core/src/controller/list/useListParams.ts

```diff
@@ -351,7 +351,7 @@ export const getQuery = ({
             ? queryFromLocation
             : hasCustomParams(params)
             ? { ...params }
-            : { filter: filterDefaultValues || {} };
```

Comment:
here, we applied filter default values only if there were no saved params. This prevented default values to be applied when a user resets all filters (because if they saved the filters once, they have some params in the store containing the sort and pagination)